doc/releases/pacific.rst

```diff
@@ -2,6 +2,253 @@
+* qa: lengthen health warning wait (`pr#50760 <https://github.com/ceph/ceph/pull/50760>`_, Patrick Donnelly)
+* qa: load file system info if not created (`pr#50923 <https://github.com/ceph/ceph/pull/50923>`_, Patrick Donnelly)
+* qa: test the "ms_mode" options in kclient workflows (`pr#50712 <https://github.com/ceph/ceph/pull/50712>`_, Jeff Layton)
+* qa: test_recovery_pool uses wrong recovery procedure (`pr#50860 <https://github.com/ceph/ceph/pull/50860>`_, Patrick Donnelly)
+* qa: wait for scrub to finish (`pr#49458 <https://github.com/ceph/ceph/pull/49458>`_, Milind Changire)
+* rbd-mirror: add information about the last snapshot sync to image status (`pr#50265 <https://github.com/ceph/ceph/pull/50265>`_, Divyansh Kamboj)
+* rbd-mirror: fix syncing_percent calculation logic in get_replay_status() (`pr#50181 <https://github.com/ceph/ceph/pull/50181>`_, N Balachandran)
+* rgw/beast: fix interaction between keepalive and 100-continue (`pr#49841 <https://github.com/ceph/ceph/pull/49841>`_, Casey Bodley, Yixin Jin)
+* rgw/coroutine: check for null stack on wakeup (`pr#49097 <https://github.com/ceph/ceph/pull/49097>`_, Casey Bodley)
+* rgw/s3: dump Message field in Error response even if empty (`pr#51200 <https://github.com/ceph/ceph/pull/51200>`_, Casey Bodley)
+* rgw: "reshard cancel" errors with "invalid argument" (`pr#49091 <https://github.com/ceph/ceph/pull/49091>`_, J. Eric Ivancich)
+* rgw: adding BUCKET_REWRITE and OBJECT_REWRITE OPS to (`pr#49095 <https://github.com/ceph/ceph/pull/49095>`_, Pritha Srivastava)
+* rgw: an empty tagset is allowed by S3 (`pr#49809 <https://github.com/ceph/ceph/pull/49809>`_, Volker Theile, Liu Lan)
+* rgw: Backport of issue 57562 to Pacific (`pr#49682 <https://github.com/ceph/ceph/pull/49682>`_, Adam C. Emerson)
+* rgw: bucket list operation slow down in special scenario (`pr#49086 <https://github.com/ceph/ceph/pull/49086>`_, zealot)
+* rgw: concurrency for multi object deletes (`pr#49327 <https://github.com/ceph/ceph/pull/49327>`_, Casey Bodley, Cory Snyder)
+* rgw: fix the problem of duplicate idx when bi list (`pr#49829 <https://github.com/ceph/ceph/pull/49829>`_, wangtengfei)
+* rgw: optimizations for handling ECANCELED errors from within get_obj_state (`pr#50886 <https://github.com/ceph/ceph/pull/50886>`_, Cory Snyder)
+* rgw: rgw_parse_url_bucket() rejects empty bucket names after 'tenant:' (`pr#50624 <https://github.com/ceph/ceph/pull/50624>`_, Casey Bodley)
+* rgw: RGWPutLC does not require Content-MD5 (`pr#49089 <https://github.com/ceph/ceph/pull/49089>`_, Casey Bodley)
+* tools/cephfs: include lost+found in scan_links (`pr#50784 <https://github.com/ceph/ceph/pull/50784>`_, Patrick Donnelly)
+* v16.2.11 (`pr#49852 <https://github.com/ceph/ceph/pull/49852>`_, Ceph Release Team)
+
+* v16.2.12 (`pr#51066 <https://github.com/ceph/ceph/pull/51066>`_, Ceph Release Team, Cory Snyder, Guillaume Abrioux)
```

Comment:
The suggestion was to remove v16.2.11 and v16.2.12 lines, not to add a blank line between them ;)

Reply:
Yes @yuriw remove lines 247, 248, and 249.